### DISCUSSION TITLE
Introduction - Getting started - Improving navigation link errors with 301

### DISCUSSION THREAD

**Discussion by JonJam** [allure-framework/allure2#2648] [Documentation] [answered]
**Reproduction steps**
1. Open the Getting Started documentation: https://allurereport.org/docs/gettingstarted-view-report/
2. Click on the Improving navigation link in the side menu

**Expected results**
Documentation is displayed

**Actual results**
Multiple 301 requests/responses that result in chrome displaying too many redirects error.

![Screenshot 2024-07-19 at 07 29 00](https://github.com/user-attachments/assets/fb5700bd-53a2-4181-9983-3814545e0d6b)

**Technical notes**
- Tried navigating to directly
- Tried in Incognito
- Tried in different browsers i.e. Firefox

**Comment by baev** [comment, marked as answer]
@JonJam, thanks. 

The issue is fixed (you may need to reload a page without the cache)

**Reply by JonJam** [reply]
Thanks @baev. Confirmed it is working now
